### ISSUE TITLE
Train tips

### ISSUE THREAD

**Issue by AndreyStille** [Janspiry/Image-Super-Resolution-via-Iterative-Refinement#51]
Hi
Thanks for your work. Can you please give advice or some tips and things to try to improve training process? 
Loss is reducing but validation results are still poor. And quality isn't improving with new epoches. Despite dataset is quite big (10000 pics) and homogeneous . I'm trying to train 256 ->1024 sr3. 
I would appreciate any tips or ideas. Thanks!

`"model": {
        "which_model_G": "sr3", // use the ddpm or sr3 network structure
        "finetune_norm": false,
        "unet": {
            "in_channel": 6,
            "out_channel": 3,
            "inner_channel": 16,
            "norm_groups": 16,
            "channel_multiplier": [
                1,
                2,
                4,
                8,
                // 8,
                // 16,
                16,
				32,
				32,
				32
            ],
            "attn_res": [
                // 16
            ],
            "res_blocks": 1,
            "dropout": 0
},`

**Comment by AndreyStille** [comment]
loss on train in logging is about 3e-2, but on inference get such results
`"beta_schedule": { // use munual beta_schedule for acceleration
            "train": {
                "schedule": "linear",
                "n_timestep": 2000,
                "linear_start": 1e-6,
                "linear_end": 1e-2
            },
            "val": {
                "schedule": "linear",
                "n_timestep": 2000,
                "linear_start": 1e-6,
                "linear_end": 1e-2
            }
        },`
![1](https://user-images.githubusercontent.com/65295663/152411242-78106ee2-894e-4a82-b340-4af45819fef5.jpg)

**Comment by Janspiry** [comment]
Hi, thanks for your attention.
You can inference the #12 .

**Comment by codgodtao** [comment]
@AndreyStille  hello! 
I have the same problem as you. The output image has a lot of noise! How do you adjust the network parameters, such as more resblock, beta_schedule or timesteps .In addition, is your data size large enough? Thank you in advance for your answer!
